script.js

```diff
@@ -0,0 +1,66 @@
+const calcRows = document.querySelector('#frame');
+
+let numEntry2 = []
```

Comment:
using arrays is fine, but since the dom values are coming in as strings, which are iterable, it probably would make it simpler to go with strings in general